### DISCUSSION TITLE
Hide curly braces within a file

### DISCUSSION THREAD

**Discussion by GizakiF** [ObsidianToAnki/Obsidian_to_Anki#603] [Q&A]
is there a way to hide or toggle curly braces on and off?

**Comment by envico801** [comment]
Hi @GizakiF are you referring to the curly braces used to create cloze deletions? These?

![cloze deletion preview](https://github.com/user-attachments/assets/2e42924d-6328-4d80-b8f6-bcafbbc9fd30)

If they are the ones I show in the image you can change their operation, with this plugin option:

![plugin option](https://github.com/user-attachments/assets/116bafc4-8ffa-4cac-a7e3-f3a11434cee6)

If it is none of the above options let me know, maybe I can help you with your problem in some other way.
